### ISSUE TITLE
Animation explaining Oppia

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#54]
It would be nice to have an animation in the prototype explaining how Oppia works (e.g. for parents who need to set up Oppia profiles for their children). Although this requires a lot of UI work, one solution to minimize complexity and maximize design iteration (and changes in the future using a tool like Illustrator) would be to integrate Lottie which seems designed for this type of use case. See PRD for the specifics of what this animation should contain and when it should show.

**Comment by BenHenning** [comment]
This is blocked on #22 for a place to hold the animation, and maybe #44 & #14 for showing this for the first time a new profile is logged into.

**Comment by BenHenning** [comment]
This is blocked on #48 for a place from which to initiate the animation, and maybe #44 & #14 for showing this for the first time a new profile is logged into.

**Comment by seanlip** [comment]
We don't need this; the onboarding flow is being redesigned.
